File: app/src/main/java/at/shockbytes/corey/storage/live/LiveWorkoutUpdateListener.java
package at.shockbytes.corey.storage.live;

import at.shockbytes.corey.workout.model.Workout;

/**
 * @author Martin Macheiner
 *         Date: 27.02.2017.
 */

public interface LiveWorkoutUpdateListener {

    void onWorkoutAdded(Workout workout);

    void onWorkoutDeleted(Workout workout);

    void onWorkoutChanged(Workout workout);

}

File: app/src/main/java/at/shockbytes/corey/storage/live/LiveWorkoutUpdateListener.kt
package at.shockbytes.corey.storage.live

import at.shockbytes.corey.common.core.workout.model.Workout

/**
 * @author Martin Macheiner
 * Date: 27.02.2017.
 */

interface LiveWorkoutUpdateListener {

    fun onWorkoutAdded(workout: Workout)

    fun onWorkoutDeleted(workout: Workout)

    fun onWorkoutChanged(workout: Workout)

}
